modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/js/components/SegmentsExperiments.es.js

```diff
@@ -75,13 +75,15 @@ function SegmentsExperiments({
 			<ClayTabs justified={true}>
 				<ClayTabs.Item
 					active={activeTab === TABS_STATES.ACTIVE}
+					className="c-pt-1"
```

Comment:
I'll change the title of this commit, from `LPS-138662 Style changes for the buttons position in the A/B Test panel` to `LPS-138662 Fix A/B Testing tab button's focus/outlined styles`
It's more accurate, but proceed as you consider, not really a big issue

Reply:
Great, I'll do it!